src/code-of-conduct.md

```diff
@@ -4,21 +4,21 @@ description: Be excellent to each other.
 toc: false
 ---
 
-As contributors and maintainers of the Dart project,
-we promise to extend courtesy and respect to everyone involved in this project
-regardless of gender, gender identity, sexual orientation, disability, age,
-race, ethnicity, religion, or level of experience.
-We expect anyone contributing to the Dart project to do the same.
+We expect Dart contributors to act professionally and respectfully, and
```

Comment:
I wonder if the right phrase here is "community members" rather than "contributors"? Could this be taken as implying that this is _our_ responsibility, rather than a responsibility we expect the entire community to uphold, perhaps?

Reply:
I'm fine with community members. Again, I was mimicking the Flutter code of conduct.